packages/react-native-app-auth/plugin/index.js

```diff
@@ -0,0 +1,17 @@
+const { withPlugins, createRunOncePlugin } = require('@expo/config-plugins');
+const { withAppAuthAppDelegate, withAppAuthAppDelegateHeader } = require('./ios');
+const { withAppAuthAppBuildGradle } = require('./android');
+
+const withAppAuth = config => {
+  return withPlugins(config, [
+    // iOS
+    withAppAuthAppDelegate,
+    withAppAuthAppDelegateHeader, // 👈 ️this one uses withDangerousMod !
+
+    // Android
+    withAppAuthAppBuildGradle, // 👈 ️this one uses withDangerousMod !
+  ]);
+};
```

Comment:
```suggestion
const withAppAuth = (config, options) => {
  return withPlugins(config, [
    // iOS
    withAppAuthAppDelegate,
    withAppAuthAppDelegateHeader, // 👈 ️this one uses withDangerousMod !

    // Android
    config => withAppAuthAppBuildGradle(config, options), // 👈 ️this one uses withDangerousMod !
  ]);
};
```

I think this is missing, otherwise 'options?.authScheme' will always be undefined.

This works great on both iOS and Android.
Thanks a lot for your work!